src/test/kotlin/io/lettuce/core/api/coroutines/CoroutinesIntegrationTests.kt

```diff
@@ -36,51 +37,39 @@ import javax.inject.Inject
  * @author Mark Paluch
  */
 @ExtendWith(LettuceExtension::class)
+@ExperimentalCoroutinesApi
```

Comment:
Let's move changes to this file into a separate commit.

Reply:
Agree, will move to a separate PR with kotlin tests pollishing